### PR TITLE
adding test for style guide

### PR DESCRIPTION
**Pull request by mmlumba**
This adds a test for style guide and fixes label issues with the checkboxes/radio buttons on the style guide:
screen shot 2017-05-01 at 10 13 42 am
screen shot 2017-05-01 at 10 14 00 am

Also adds a different ID to each of the searchable dropdowns:
screen shot 2017-05-01 at 10 14 21 am

### CODE REVIEW

**Comment by gnakm** [comment]
@mmlumba I made a tweak in this issue that effects this PR
https://github.com/department-of-veterans-affairs/caseflow/issues/1078

The spacing underneath `<h3>` should be 20px. Could you verify that this change was made.

**Comment by mmlumba** [comment]
@gnakm does this look better?
screen shot 2017-05-01 at 1 00 20 pm

**Comment by gnakm** [comment]
👍 👍 👍 lgtm!

File: spec/feature/styleguide_spec.rb [hunk 1]
```diff
@@ -1,11 +1,8 @@
 require "rails_helper"
 
-# Future Style Guide Test...remove skip after fixing accessibility errors
-# :nocov:
 RSpec.feature "Style Guide" do
-  skip "renders and is accessible" do
+  scenario "renders and is accessible" do
```
Comment (shanear) [inline spec/feature/styleguide_spec.rb]: tumblr_mekm2ryfrg1r7j4bg

**Pull request merged**